Some examples from openhab2-jython/Script Examples/hello_world.py don't work

**Describe the bug**
Some examples on the page openhab2-jython/Script Examples/hello_world.py don't work.

**To Reproduce**
Steps to reproduce the behavior:
1. Use Openhab 2.4, Jython 2.7.0 standalone in a docker environment
2. Follow the instructions on the wiki to enable Jython support
3. Run the hello_world.py examples
4. Error 1

**- RawAPICron**
Error: 

> 10-May-2019 22:30:02.680 [ERROR] [ion.module.script.internal.ScriptEngineManagerImpl] - Error during evaluation of script 'file:/openhab/conf/automation/jsr223/personal/in_test/hello_world.py': TypeError: can't convert ['Test tag'] to java.util.Set in <script> at line number 24

5. Error 2:
self.log.info("message") doesn't work for
- RawAPICron
- RawAPIStateUpdate
- ExtensionCron
- ExtensionItemUpdate

**Expected behavior**
The self.log.info works.

When using the @rule decorator, the problem isn't there.

**Environment (please complete the following information):**
 - Jython version: [e.g. 2.7.0 standalone,]
 - openHAB version: [e.g. 2.4.0 release build]
 - Release: latest master (downloaded 10-05-2019, around 22:00 amsterdam time)

**Additional context**
Log entry:

> 10-May-2019 22:40:00.005 [DEBUG] [.smarthome.automation.core.internal.RuleEngineImpl] - The trigger 'Hello_World_Cron_Trigger' of rule '10b8239a-dcd8-4772-9ca9-a5dcb6637784' is triggered.
10-May-2019 22:40:00.008 [ERROR] [.smarthome.automation.core.internal.RuleEngineImpl] - Failed to execute rule '10b8239a-dcd8-4772-9ca9-a5dcb6637784': Fail to execute action: 1
10-May-2019 22:40:00.010 [DEBUG] [.smarthome.automation.core.internal.RuleEngineImpl] -
java.lang.RuntimeException: Fail to execute action: 1
        at org.eclipse.smarthome.automation.core.internal.RuleEngineImpl.executeActions(RuleEngineImpl.java:1207) ~[205:org.eclipse.smarthome.automation.core:0.10.0.oh240]
        at org.eclipse.smarthome.automation.core.internal.RuleEngineImpl.runRule(RuleEngineImpl.java:1004) [205:org.eclipse.smarthome.automation.core:0.10.0.oh240]
        at org.eclipse.smarthome.automation.core.internal.TriggerHandlerCallbackImpl$TriggerData.run(TriggerHandlerCallbackImpl.java:91) [205:org.eclipse.smarthome.automation.core:0.10.0.oh240]
        at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) [?:?]
        at java.util.concurrent.FutureTask.run(FutureTask.java:266) [?:?]
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) [?:?]
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) [?:?]
        at java.lang.Thread.run(Thread.java:748) [?:?]
Caused by: org.python.core.PyException
        at org.python.core.Py.AttributeError(Py.java:205) ~[?:?]
        at org.python.core.PyObject.noAttributeError(PyObject.java:1013) ~[?:?]
        at org.python.core.PyObject.__getattr__(PyObject.java:1008) ~[?:?]
        at org.python.pycode._pyx50.execute$3(<script>:23) ~[?:?]
        at org.python.pycode._pyx50.call_function(<script>) ~[?:?]
        at org.python.core.PyTableCode.call(PyTableCode.java:167) ~[?:?]
        at org.python.core.PyBaseCode.call(PyBaseCode.java:307) ~[?:?]
        at org.python.core.PyBaseCode.call(PyBaseCode.java:198) ~[?:?]
        at org.python.core.PyFunction.__call__(PyFunction.java:482) ~[?:?]
        at org.python.core.PyMethod.instancemethod___call__(PyMethod.java:237) ~[?:?]
        at org.python.core.PyMethod.__call__(PyMethod.java:228) ~[?:?]
        at org.python.core.PyMethod.__call__(PyMethod.java:218) ~[?:?]
        at org.python.core.PyMethod.__call__(PyMethod.java:213) ~[?:?]
        at org.python.core.PyObject._jcallexc(PyObject.java:3626) ~[?:?]
        at org.python.core.PyObject._jcall(PyObject.java:3658) ~[?:?]
        at org.python.proxies.__builtin__$RawAPICron$55.execute(Unknown Source) ~[?:?]
        at org.eclipse.smarthome.automation.module.script.rulesupport.shared.simple.SimpleRuleActionHandlerDelegate.execute(SimpleRuleActionHandlerDelegate.java:34) ~[?:?]
        at org.eclipse.smarthome.automation.module.script.rulesupport.internal.delegates.SimpleActionHandlerDelegate.execute(SimpleActionHandlerDelegate.java:60) ~[?:?]
        at org.eclipse.smarthome.automation.core.internal.RuleEngineImpl.executeActions(RuleEngineImpl.java:1199) ~[?:?]
        ... 7 more

--- a/Script Examples/hello_world.py
+++ b/Script Examples/hello_world.py
@@ -1,37 +1,122 @@
 from core.rules import rule
-from core.triggers import when, CronTrigger
+from core.triggers import when
 
+@rule("Jython Hello World (cron decorators)", description="This is an example cron triggered rule using decorators", tags=["Test tag", "Hello World"])# [description and tags are optional]
+@when("Time cron 0/10 * * * * ?")
+def hellowWorldCronDecorators(event):
+    hellowWorldCronDecorators.log.info("Hello World!")
+
+# The following cron triggered rules are for demonstration of how the Jython helper libraries have evolved
 '''
+from core.triggers import CronTrigger
+from core.rules import rule
+
+@rule("Jython Hello World (CronTrigger extension with rule decorator)", description="This is an example rule using a CronTrigger extension and rule decorator", tags=["Example rule tag"])
+class CronExtensionWithRuleDecorator(object):
+    def __init__(self):
+        self.triggers = [CronTrigger("0/10 * * * * ?").trigger]
+
+    def execute(self, module, inputs):
+        self.log.info("Hello World!")
+'''
+'''
+from core.triggers import CronTrigger
+from core.log import logging, LOG_PREFIX
+
 scriptExtension.importPreset("RuleSupport")
 scriptExtension.importPreset("RuleSimple")
 
-class RawAPICron(SimpleRule):
+class CronTriggerExtension(SimpleRule):
+    def __init__(self):
+        self.triggers = [CronTrigger("0/10 * * * * ?").trigger]
+        self.name = "Jython Hello World (CronTrigger extension)"
+        self.description = "This is an example Jython rule using a CronTrigger extension"
+        self.tags = set("Example rule tag")
+        self.log = logging.getLogger("{}.Hello World (CronTrigger extension)".format(LOG_PREFIX))
+
+    def execute(self, module, inputs):
+        self.log.info("Hello World!")
+
+automationManager.addRule(CronTriggerExtension())
+'''
+'''
+from org.slf4j import Logger, LoggerFactory
+
+scriptExtension.importPreset("RuleSupport")
+scriptExtension.importPreset("RuleSimple")
+
+class GenericCronTriggerRawAPI(SimpleRule):
     def __init__(self):
         self.triggers = [
             TriggerBuilder.create()
-                    .withId("Hello_World_Cron_Trigger")
+                    .withId("Hello_World_Cron_Trigger")# no spaces akkowed in trigger ID
                     .withTypeUID("timer.GenericCronTrigger")
                     .withConfiguration(
                         Configuration({
                             "cronExpression": "0/10 * * * * ?"
                         })).build()
         ]
-        self.name = "Hello world cron rule (raw API)"
-        self.description = "This is an example cron rule using the raw API"
-        self.tags = [ "Test tag" ]
+        self.name = "Jython Hello World (GenericCronTrigger raw API)"
+        self.description = "This is an example Jython cron rule using the raw API"
+        self.tags = set("Example rule tag")
+        self.log = LoggerFactory.getLogger("jython.Hello World (GenericCronTrigger raw API)")
 
     def execute(self, module, inputs):
-        self.log.info("This is a 'hello world!' from a Jython rule (raw API): Cron")
+        self.log.info("Hello World!")
 
-automationManager.addRule(RawAPICron())
+automationManager.addRule(GenericCronTriggerRawAPI())
 '''
 
+# These rules require an Item names Test_Switch_1
 '''
+from core.rules import rule
+from core.triggers import when
+
+@rule("Jython Hello World (Item update decorators)", description="This is an example Item update triggered rule using decorators", tags=["Example rule tag"])
+@when("Item Test_Switch_1 received update")
+def hellowWorldItemUpdateDecorators(event):
+    hellowWorldItemUpdateDecorators.log.info("Hello World!")
+'''
+# The following Item update triggered rules are for demonstration of how the Jython helper libraries have evolved
+'''
+from core.triggers import ItemStateUpdateTrigger
+from core.rules import rule
+
+@rule("Jython Hello World (ItemStateUpdateTrigger extension with rule decorator)", description="This is an example rule using a ItemStateUpdateTrigger extension and rule decorator", tags=["Example rule tag"])
+class ItemStateUpdateTriggerExtensionWithRuleDecorator(object):
+    def __init__(self):
+        self.triggers = [ItemStateUpdateTrigger("Test_Switch_1").trigger]
+    
+    def execute(self, module, inputs):
+        self.log.info("Hello World!")
+'''
+'''
+from core.triggers import ItemStateUpdateTrigger
+from core.log import logging, LOG_PREFIX
+
 scriptExtension.importPreset("RuleSupport")
 scriptExtension.importPreset("RuleSimple")
 
-# make sure you have an item named Test_Switch_1, or swap out the item for one that already exists
-class RawAPIStateUpdate(SimpleRule):
+class ItemStateUpdateTriggerExtension(SimpleRule):
+    def __init__(self):
+        self.triggers = [ItemStateUpdateTrigger("Test_Switch_1").trigger]
+        self.name = "Jython Hello World (ItemStateUpdateTrigger extension)"
+        self.description = "This is an example rule using an ItemStateUpdateTrigger extension"
+        self.tags = set("Example rule tag")
+        self.log = logging.getLogger("{}.Hello World (ItemStateUpdateTrigger extension)".format(LOG_PREFIX))
+
+    def execute(self, module, inputs):
+        self.log.info("Hello World!")
+
+automationManager.addRule(ItemStateUpdateTriggerExtension())
+'''
+'''
+from org.slf4j import Logger, LoggerFactory
+
+scriptExtension.importPreset("RuleSupport")
+scriptExtension.importPreset("RuleSimple")
+
+class ItemStateUpdateTriggerRawAPI(SimpleRule):
     def __init__(self):
         self.triggers = [
             TriggerBuilder.create()
@@ -43,91 +128,13 @@ class RawAPIStateUpdate(SimpleRule):
                     }))
                 .build()
         ]
-        self.name = "Hello world item state update rule (raw API)"
+        self.name = "Jython Hello World (ItemStateUpdateTrigger raw API)"
+        self.description = "This is an example Jython cron rule using the raw API"
+        self.tags = set("Example rule tag")
+        self.log = logging.getLogger("{}.Hello World (ItemStateUpdateTrigger raw API)".format(LOG_PREFIX))
 
     def execute(self, module, input):
-        self.log.info("This is a 'hello world!' from a Jython rule (raw API): ItemStateUpdateTrigger")
+        self.log.info("Hello World!")
 
-automationManager.addRule(RawAPIStateUpdate())
-'''
-
-'''
-from core.triggers import CronTrigger
-
-scriptExtension.importPreset("RuleSupport")
-scriptExtension.importPreset("RuleSimple")
-
-# requires CronTrigger import
-class ExtensionCron(SimpleRule):
-    def __init__(self):
-        self.triggers = [ CronTrigger("0/10 * * * * ?").trigger ]
-        self.name = "Hello world cron rule (extension)"
-    
-    def execute(self, module, inputs):
-        self.log.info("This is a 'hello world!' from a Jython rule (extension): CronTrigger")
-
-automationManager.addRule(ExtensionCron())
-'''
-
-'''
-from core.triggers import ItemStateUpdateTrigger
-
-scriptExtension.importPreset("RuleSupport")
-scriptExtension.importPreset("RuleSimple")
-
-class ExtensionItemUpdate(SimpleRule):
-    def __init__(self):
-        self.triggers = [ ItemStateUpdateTrigger("Test_Switch_1").trigger ]
-        self.name = "Hello world item state update rule (extension)"
-    
-    def execute(self, module, inputs):
-        self.log.info("This is a 'hello world!' from a Jython rule (extension): ItemStateUpdateTrigger")
-
-automationManager.addRule(ExtensionItemUpdate())
-'''
-
-'''
-from core.triggers import CronTrigger
-from core.rules import rule
-
-@rule("Hello world cron rule (extension with rule decorator)")
-class ExtensionCronWithRule(object):
-    def __init__(self):
-        self.triggers = [ CronTrigger("0/10 * * * * ?").trigger ]
-        self.name = "Hello world cron rule (extension with rule decorator)"
-    
-    def execute(self, module, inputs):
-        self.log.info("This is a 'hello world!' from a Jython rule (extension): CronTrigger")
-'''
-
-'''
-from core.triggers import ItemStateUpdateTrigger
-from core.rules import rule
-
-@rule("Hello world item update rule (extension with rule decorator)")
-class ExtensionItemUpdateWithRule(object):
-    def __init__(self):
-        self.triggers = [ ItemStateUpdateTrigger("Test_Switch_1").trigger ]
-        self.name = "Hello world item update rule (extension with rule decorator)"
-    
-    def execute(self, module, inputs):
-        self.log.info("This is a 'hello world!' from a Jython rule (extension with rule): ItemUpdateTrigger")
-'''
-
-from core.rules import rule
-from core.triggers import when
-
-@rule("Hello World cron rule (decorator)", description="This is an example rule that demonstrates using a cron rule with decorators", tags=["Test tag", "Hello World"])# [description and tags are optional]
-@when("Time cron 0/10 * * * * ?")
-def hellowWorldDecoratorCron(event):
-    hellowWorldDecoratorCron.log.info("This is a 'hello world!' from a Jython rule (decorator): Cron")
-
-'''
-from core.rules import rule
-from core.triggers import when
-
-@rule("Hello World item update rule (decorator)")
-@when("Item Test_Switch_1 received update")
-def hellowWorldDecoratorItemUpdate(event):
-    hellowWorldDecoratorItemUpdate.log.info("This is a 'hello world!' from a Jython rule (decorator): Item update")
+automationManager.addRule(ItemStateUpdateTriggerRawAPI())
 '''

